mne/viz/montage.py

```diff
@@ -8,7 +8,7 @@
 
 @verbose
 def plot_montage(montage, scale_factor=20, show_names=True, kind='topomap',
-                 show=True, sphere=None, verbose=None):
+                 show=True, sphere=None, verbose=None, axes=None):
```

Comment:
```suggestion
                 show=True, sphere=None, axes=None, *, verbose=None):
```

verbose must always come last.  You'll need to update the order of docstring parameter entries too.

Reply:
I was worried about backward compatibility with this. Making it a kwarg-only attribute sounds great. So I'll also implement that change of making `verbose` a kwarg in these functions?

Reply:
yes. We're sort of in transition to making wider use of kwarg-only parameters. `verbose` is one that should always be kwarg-only; others are sort of case-by-case and for now are mostly being done in functions/methods with really long parameter lists.